### PR TITLE
chore: setup-java v2

### PR DESCRIPTION
**Pull request by sullis**

### CODE REVIEW

**Comment by sberyozkin** [comment]
@sullis thanks, LGTM, but lets wait for Roberto to review

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -38,10 +38,11 @@ jobs:
       - uses: actions/checkout@v2
         name: checkout
 
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         name: set up jdk ${{matrix.java}}
         with:
           java-version: ${{matrix.java}}
+          distribution: 'zulu'
```
Comment (radcortez) [inline .github/workflows/build.yml:45]: Since we are moving to `2`, I would probably change the dist to be `adopt`.

**Review by radcortez — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (sullis) [reply .github/workflows/build.yml]: Done.

**Comment by sullis** [comment]
fyi:

image

https://github.com/actions/setup-java

**Comment by radcortez** [comment]
> fyi: setup-java recommends using temurin instead of adopt
> 
> image
> 
> https://github.com/actions/setup-java

Sure.

**Review by radcortez — APPROVED** [review]

**Comment by sberyozkin** [comment]
@sullis @radcortez thanks

**Pull request merged**